docs/install_guides/installing-openvino-raspbian.md

```diff
@@ -32,33 +32,68 @@
 
 ## <a name="install-openvino"></a>Step 1: Download and Install OpenVINO Runtime
 
+@sphinxdirective
+
 1. Open the Terminal or your preferred console application.
 2. Create an installation folder for OpenVINO. If the folder already exists, skip this step.
-   ```sh
-   sudo mkdir -p /opt/intel
-   ```
-   > **NOTE**: The `/opt/intel` path is the recommended folder path for administrators or root users. If you prefer to install OpenVINO in regular userspace, the recommended path is `/home/<USER>/intel`. You may use a different path if desired.
-
-3. Go to your `~/Downloads` directory and download OpenVINO Runtime archive file for Debian from [OpenVINO package repository](https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.2/linux/).
-   ```sh
-   cd ~/Downloads/
-   sudo wget https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.2/linux/l_openvino_toolkit_debian9_arm_2022.2.0.7713.af16ea1d79a_armhf.tgz -O openvino_2022.2.0.7713.tgz
-   ```
+
+   .. code-block:: sh
+
+      sudo mkdir -p /opt/intel
+   
+   .. note::
+   
+      The `/opt/intel` path is the recommended folder path for administrators or root users. If you prefer to install OpenVINO in regular userspace, the recommended path is `/home/<USER>/intel`. You may use a different path if desired.
+
+3. Go to your `~/Downloads` directory and download OpenVINO Runtime archive file for Debian from `OpenVINO package repository <https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.3/linux/>`_.
+
+   .. tab:: ARM 32-bit
+
+      .. code-block:: sh
+
+         cd ~/Downloads/
+         sudo wget https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.3/linux/l_openvino_toolkit_debian9_arm_2022.3.0.<update>.<patch>_armhf.tgz -O openvino_2022.3.0.tgz
```

Comment:
Please change name of archive to 
l_openvino_toolkit_debian9_2022.3.0.<update>.<patch>_armhf.tgz